docs/event-import.md

```diff
@@ -0,0 +1,57 @@
+
+## Import status
+
+After the import process is initiated, it runs asynchronously. The endpoints return an event import ID that can be used to check the status of the import at a later time. Here are the endpoints you can utilize to check the status of your imports
+
+- [Retrieve a list of event imports](https://api-docs.scarf.sh/v2.html#tag/External-event-import/operation/getEventImports)
+- [Retrieve the import log for an event import](https://api-docs.scarf.sh/v2.html#tag/External-event-import/operation/getImportLogs)
+- [Retrieve a specific event import](https://api-docs.scarf.sh/v2.html#tag/External-event-import/operation/getEventImport)
+ 
+## Cancel an import
+
+To cancel an ongoing import process, use the [Abort event import](https://api-docs.scarf.sh/v2.html#tag/External-event-import/operation/abortEventImport) endpoint.
+
+## Import example
+
+Here's an example of importing events associated to a package.
+In this example, the name of this file is `events.ndjson`
+```ndjson
+{"$package":"970493a1-4ca0-4a4d-a085-fdce578e5a08","$remote_address":"152.241.796.177","$time":"2024-06-04T00:00:00Z","$unique_id":"c20b1271-fb3f-abfa-df12-ef3cda4b2aa0"}
+{"$package":"970493a1-4ca0-4a4d-a085-fdce578e5a08","$remote_address":"600.188.717.651","$time":"2024-06-01T00:00:00Z","$unique_id":"9053a19a-15a9-3695-bd37-b055a45949c1"}
+{"$package":"970493a1-4ca0-4a4d-a085-fdce578e5a08","$remote_address":"665.921.984.205","$time":"2024-06-25T00:00:00Z","$unique_id":"09b5b69a-0af0-8002-2c2b-39df3d5685a4"}
+```
+
+Then, this is the example import script
+```bash
+#!/usr/bin/env bash
+
+curl -v \
+  -H "Authorization: Bearer {token}" \
+  -H "Content-Type: application/json" \
+  -X POST https://api.scarf.sh/v2/{owner}/import
+  --data-binary @events.ndjson
+```
+
+To re-iterate the docs regarding the [$unique_id](https://api-docs.scarf.sh/v2.html#tag/External-event-import/operation/importEvents) field,
+
+> Unique identifier that uniquely identifies the event and is used for de-duplication purposes, if provided.
+
+If the same value for the `$unique_id` field is used, only the first event will be processed and the rest will be ignored. If `$unique_id` is not provided, we will generate one internally.
+
+## Automation
+
+Currently, there is a limit of 15 concurrent imports but there are no limits on how many lines of events per import. A rough estimate of an import that has 1000 events takes about 1.2 minutes when the system is not under heavy load. So if you have an automation with our import api, it would be ideal to set a sleep interval after the concurrent import limit based on the estimated 1 import duration, the sleep interval ideally should have an exponential backoff. This means that if you already have 15 imports in progress, you should schedule the next import after 1.2 minutes.
```

Comment:
I am not sure we should give away the details on the timings here. I don't think we want to suggest any guarantees to users when it comes to timings. Would be happy if we'd be more vague here.

Reply:
thanks for the feedback!